themes/classic/templates/catalog/_partials/product-prices.tpl

```diff
@@ -2,37 +2,41 @@
   <div class="product-prices">
     {block name='product_discount'}
       {if $product.has_discount}
-        <p class="product-discount">
+        <div class="product-discount">
           {hook h='displayProductPriceBlock' product=$product type="old_price"}
           <span class="regular-price">{$product.regular_price}</span>
-        </p>
+        </div>
       {/if}
     {/block}
 
     {block name='product_price'}
-      <p
+      <div
         class="product-price h5 {if $product.has_discount}has-discount{/if}"
         itemprop="offers"
         itemscope
         itemtype="https://schema.org/Offer"
       >
         <link itemprop="availability" href="https://schema.org/InStock"/>
         <meta itemprop="priceCurrency" content="{$currency.iso_code}">
-        <span itemprop="price" content="{$product.price_amount}">{$product.price}</span>
-        {if $configuration.display_taxes_label}
-          <small class="text-capitalize">{$product.labels.tax_short}</small>
-        {/if}
 
-        {hook h='displayProductPriceBlock' product=$product type="price"}
+        <div class="current-price">
+          <span itemprop="price" content="{$product.price_amount}">{$product.price}</span>
 
-        {if $product.has_discount}
-          {if $product.discount_type === 'percentage'}
-            <span class="discount discount-percentage">{l s='Save %percentage%' d='Shop.Theme.Catalog' sprintf=['%percentage%' => $product.discount_percentage]}</span>
-          {else}
-            <span class="discount discount-amount">{l s='Save %amount%' d='Shop.Theme.Catalog' sprintf=['%amount%' => $product.discount_amount]}</span>
+          {if $product.has_discount}
+            {if $product.discount_type === 'percentage'}
+              <span class="discount discount-percentage">{l s='Save %percentage%' d='Shop.Theme.Catalog' sprintf=['%percentage%' => $product.discount_percentage]}</span>
+            {else}
+              <span class="discount discount-amount">{l s='Save %amount%' d='Shop.Theme.Catalog' sprintf=['%amount%' => $product.discount_amount]}</span>
+            {/if}
           {/if}
-        {/if}
-      </p>
+        </div>
+
+        {block name='product_unit_price'}
+          {if $displayUnitPrice}
```

Comment:
It would be so nice to have it in `$configuration` like `$configuration.display_taxes_label`.